src/sage/schemes/jacobians/abstract_jacobian.py

```diff
@@ -8,40 +8,23 @@
 - William Stein (2005)
 
 """
-#*****************************************************************************
+
+# ****************************************************************************
 #       Copyright (C) 2005 William Stein <wstein@gmail.com>
 #
-#  Distributed under the terms of the GNU General Public License (GPL)
-#  as published by the Free Software Foundation; either version 2 of
-#  the License, or (at your option) any later version.
-#                  http://www.gnu.org/licenses/
-#*****************************************************************************
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#                  https://www.gnu.org/licenses/
+# ****************************************************************************
+
+from sage.categories.schemes import Jacobians
 from sage.categories.fields import Fields
-_Fields = Fields()
 from sage.schemes.generic.scheme import Scheme, is_Scheme
 from sage.structure.richcmp import richcmp_method, richcmp
 
-
-def is_Jacobian(J):
```

Comment:
Does it need deprecation before removal?

Reply:
I guess that that is not really used.

But to be safe, I recovered it and deprecated.